cnf/mkversionheader.sh

```diff
@@ -4,11 +4,12 @@ TMP="$1".tmp
 DST="$1"
 
 # Determine build version and date
-GAP_BUILD_VERSION=`git describe --tags --dirty || echo`
-GAP_BUILD_DATE=`date +"%Y-%m-%d %H:%M:%S (%Z)"`
-if test x"${GAP_BUILD_VERSION}" = x ; then
+if test -d ../.git ; then
```

Comment:
This test is wrong. The path `../.git` is taken relative to the to current working directory (cwd / pwd), not relative to the script. But the script is invoked from the Makefile, which is in the GAP root directory.